service/src/main/java/tds/config/endpoints/ConfigController.java

```diff
@@ -0,0 +1,48 @@
+package tds.config.endpoints;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import tds.config.ClientSystemFlag;
+import tds.config.ClientTestProperty;
+import tds.config.services.ConfigService;
+import tds.config.web.resources.ClientSystemFlagResource;
+import tds.config.web.resources.ClientTestPropertyResource;
+
+
+@RestController
+@RequestMapping("/config")
+public class ConfigController {
+    private static final Logger LOG = LoggerFactory.getLogger(ConfigController.class);
+    private final ConfigService configService;
+
+    @Autowired
+    public ConfigController(ConfigService configService) {
+        this.configService = configService;
+    }
+
+    @RequestMapping(value = "/isAlive", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
+    @ResponseBody
```

Comment:
I think we will probably end up with a shared jar for the microservices containing a small bit of shared code.  I'm not 100% sure if this is a good case for that without seeing how other services flush out.

To your comment, I think rather than a BaseController we would instead have a simple @RestController that "/isAlive" implemented.  No need to extend it otherwise everything that extended it would have an isAlive() method via the parent.